WireUI/Package.swift

```diff
@@ -108,7 +107,6 @@ let package = Package(
             name: "WireViewsDebugUI",
             dependencies: [
                 "WireAuthenticationUI",
-                .product(name: "WireDomainPackage", package: "WireDomainPackage"),
```

Comment:
`WireDomainPkg` is removed as dependency of `WireUI` since it is linked into `WireDomain` already. It could lead to implicitly duplicated symbols in the final executable.

Instead the files for the types needed are symlinked into the `WireSettingsUI` target and adapters in the app target bridge between the two types.